### PR TITLE
chore: prepare for 3.9.0 release.

### PR DESCRIPTION
**Pull request by ozayr-zaviar**
## Summary
- prepare for release 3.9.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
 # Optimizely Ruby SDK Changelog
 
+## 3.9.0
+September 16, 2021
+
+### New Features:
+- Add new public properties to OptimizelyConfig. ([#285](https://github.com/optimizely/ruby-sdk/pull/285))
```
Comment (jaeopt) [inline CHANGELOG.md:7]:
```suggestion
- Add new public properties to `OptimizelyConfig`. ([#285](https://github.com/optimizely/ruby-sdk/pull/285))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+	- sdkKey
+ 	- environmentKey
+	- attributes
+	- audiences
+	- events
+	- experimentRules and deliveryRules to `OptimizelyFeature`
+	- audiences to `OptimizelyExperiment`
+- For details, refer to our documentation page: [https://docs.developers.optimizely.com/full-stack/v4.0/docs/optimizelyconfig-ruby](https://docs.developers.optimizely.com/full-stack/v4.0/docs/optimizelyconfig-ruby).
+
+### Deprecated
```
Comment (jaeopt) [inline CHANGELOG.md:17]:
```suggestion
### Deprecated:
```

**Review by The-inside-man — APPROVED** [review]
LGTM

**Review by jaeopt — APPROVED** [review]
LGTM with a couple of nits

**Pull request merged**